Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 <Project>
     <PropertyGroup>
-        <AspNetCoreVersion>5.0.0-preview.7.20326.3</AspNetCoreVersion>
-        <BlazorVersion>5.0.0-preview.7.20326.3</BlazorVersion>
-        <EntityFrameworkVersion>5.0.0-preview.7.20326.1</EntityFrameworkVersion>
-        <SystemNetHttpJsonVersion>5.0.0-preview.7.20326.1</SystemNetHttpJsonVersion>
+        <AspNetCoreVersion>5.0.0.0</AspNetCoreVersion>
```

Comment:
Shouldn't these just be 5.0.0? I don't think we ever use a fourth digit in our package version numbers.